sample_workloads/lit-gpt-demo/README.md

```diff
@@ -0,0 +1,363 @@
+
+`jobTimestamp `needs to be a timestamp or a unique identifier.
+
+`experimentDir `refers to a directory in the GCS bucket (specified by `gcsBucket`) for logging. In the example above,` pir-pythia-6.9b/training_logs/` is a directory already set up for logging in the shared GCS bucket `litgpt-public-bucket`.  \
+Alternatively, you can create your own directory (e.g. named `logDir`) under your own GCS bucket (e.g. named `myBucket`). Then the logs will be saved at the target location (e.g. `gs://myBucket`/`logDir`) designated by the parameter `experimentDir`.
+
+`dataDir `refers to a directory in the GCS bucket (specified by `gcsBucket`) for training data. In the example above, `openwebtext_dataset` is a directory containing the training data in the shared GCS bucket `litgpt-public-bucket`. \
+Alternatively, you can create your own directory (e.g. named `dataSourceDir`) under your own GCS bucket (e.g. named `myBucket`). Then the training data will be loaded from the source location (e.g. `gs://myBucket/dataSourceDir/train.bin` and `gs://myBucket/dataSourceDir/val.bin`) designated by the parameter `dataDir`.
+
+`image `refers to the Docker image set up for LitGPT. The value of this flag is in the following format:  `<repository_location>-docker.pkg.dev/<project>/<repository_name>/litgpt-full:<tag>`
+
+**Note:** `<repository_location>`  and` <repository_name>` can be found in section [Setup Artifact Registry](#setup-artifact-registry) above. `&lt;project>` name can be found in the section [Environment Setup](#environment-setup) above. `&lt;tag>` can be found in the section [Setup Docker files and Scripts](#setup-docker-files-and-scripts) above.
+
+Running lit-gpt without the modifications to helm file <code>values.yaml</code> will not work!</strong>
+
+**Note:** before running the command above again, either do `helm uninstall &lt;HELM_EXPERIMENT_NAME>` to fully erase your previous Helm experiment and free the cluster nodes or change the experiment name to a new one so both experiments are kept.
+
+You can check the status of your workload via any of the following:
+
+**Note:** pod0 contains logs that other pods in the same experiment do not contain.
+
+
+### MFU Calculation 
+
+MFU can be calculated by consuming the `metrics.csv` file output by LitGPT. During a training run this file can be found in the litgpt container at `/workspace/out/openwebtext/version_0/metrics.csv` . After training is completed this file will be uploaded as part of the `experimentDir` specified in the helm values.
+
+Step times are presented in the csv as aggregate times in the column `time/train`, so the value used should be 
+`time/train[n] - time/train[n-1]` .
+ 
+
+MFU for this sample workload can be calculated using the formula: 
+
+` ( 6 * (6 * 8 * 40 ) * 4096 * 7e10) / (steptime * 8 * 40 * 1.979e15 / 2 ) = MFU  `
```

Comment:
I think there should be more explanation on what each of these numbers are. Maybe instead, give a general formula for the equation and we could show an example.